File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/nursing/shared/dto/nursing-opd-checkout.dto.ts
import { Visit } from "../../../appointments/shared/visit.model";
import { FinalDiagnosis_DTO } from "./final-diagnosis.dto";

export class NursingOpdCheckOut_DTO {
    public PatientVisitId: number = 0;
    public PatientId: number = 0;
    public PerformerId: number = 0;
    public DepartmentId: number = 0;
    public PerformerName: string = '';
    public DepartmentName: string = '';
    public FollowUpDay: number = 0;
    public ConcludedNote: string = '';
    public DiagnosisList: Array<FinalDiagnosis_DTO> = [];
    public Visit: Visit = new Visit;


}

File: Code/Websites/DanpheEMR/Services/Nursing/DTOs/NursingOpdCheckOut_DTO.cs
using DanpheEMR.Controllers.Nursing.DTOs;
using DanpheEMR.ServerModel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DanpheEMR.Services.Nursing.DTOs
{
    public class NursingOpdCheckOut_DTO
    {
        public int PatientVisitId { get; set; }
        public int PatientId { get; set; }
        public int PerformerId { get; set; }
        public int DepartmentId { get; set; }
        public string PerformerName { get; set; }
        public string DepartmentName { get; set; }
        public int FollowUpDay { get; set; }
        public string ConcludedNote { get; set; }

        public DateTime ConcludedDate { get; set; }
        public List<AddFinalDiagnosis_DTO> DiagnosisList { get; set; }
        public VisitModel Visit { get; set; }



    }
}
